docs/en/Running-Alluxio-on-Mesos.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: global
+title: Running Alluxio on Mesos
+nickname: Alluxio on Mesos
+group: Deploying Alluxio
+priority: 4
+---
+
+Alluxio can be deployed through Mesos. This allows Mesos to manage the resources used by Alluxio.
```

Comment:
Done, and also updated the code to properly calculate how much memory is needed by an Alluxio worker